examples/nidcpower_source_dc_voltage/README.md

```diff
@@ -27,12 +27,20 @@ This is a MeasurementLink example that sources and measures a DC voltage with an
 
 - NI-DCPower
 
+By default, this example uses a physical instrument. To use a simulated instrument, follow the steps below:
```

Comment:
Move this content back to "Required Hardware". Don't move it to "Required Driver Software".